src/pages/about/about.js

```diff
@@ -82,6 +97,80 @@ export default function About() {
           </div>
         </div>
       </section>
+
+      <section className="section-four">
+        <div className="form-background">
+          <div className="form-title">
+            <p id="title-text">Contact us</p>
+          </div>
+          <div >
+          <form onSubmit={handleSubmit}>
+            <div className="first-input-row">
+                <div className="field">
+                    <label className="label">First Name*</label>
+                    <div>
+                      <input
+                        className="small-input"
+                        name="firstName"
+                        value={formdata.firstName}
+                        onChange={handleChange}
+                      />
+                    </div>
+                  </div>
+                  <div className="field">
+                    <label className="label">Last Name*</label>
+                    <div >
+                      <input
+                        className="small-input"
+                        name="lastName"
+                        value={formdata.lastName}
+                        onChange={handleChange}
+                      />
+                    </div>
+                </div>
+              </div>
+              <div className="field">
+                <label className="label">Email*</label>
+                <div>
+                  <input
+                    className="large-input"
+                    name="email"
+                    value={formdata.email}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div className="field">
+                <label className="label">Phone</label>
+                <div >
+                  <input
+                    className="large-input"
+                    name="phone"
+                    value={formdata.phone}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div className="field">
+                <label className="label">Message</label>
+                <div >
+                  <textarea
+                    className="message-input"
+                    name="message"
+                    value={formdata.message}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div >
+                <button className="submit-button" type="submit">
+                  Send
+                </button>
+              </div>
+            </form>
+          </div>
+        </div>
+      </section>
 	</>
 	);
 };
```

Comment:
It looks like your indentation is set to 2 spaces here. Please check the settings in your IDE to make sure it is set to 4 spaces to match the style guide. If we are all using different spacing and someone else changes that spacing when they pull in your merged code, then it will show up in their next pull request as changes. This adds to the amount of code that looks like it needs to be reviewed, when really it is just white space changes.